odd-collector/odd_collector/adapters/postgresql/utils.py

```diff
@@ -0,0 +1,32 @@
+from adapters.postgresql.models import UniqueConstraint, Table
+
+
+def group_uniques_constraints_by_table(
+        unique_constraints: list[UniqueConstraint],
+) -> dict[str, UniqueConstraint]:
+    """
```

Comment:
Function can be deleted, as it is not being used.
We use function in `mappers/relationships/utils.py` for this purpose, because it provides functionality where for one table name we can have a list of `UniqueConstraint`s.